data/CMakeLists.txt

```diff
@@ -24,22 +24,24 @@ set(install_files
 	${CMAKE_CURRENT_BINARY_DIR}/scripts.zip
 	${CMAKE_CURRENT_BINARY_DIR}/backgrounds.zip
 	${CMAKE_CURRENT_BINARY_DIR}/rules.zip
-	api.lua
-	bot_api.lua
-	rules_api.lua
-	Icon.bmp
-	config.xml
-	inputconfig.xml
-	server.xml
-	lang_cs.xml
-	lang_de.xml
-	lang_en.xml
-	lang_es.xml
-	lang_fr.xml
-	lang_it.xml)
+	${CMAKE_CURRENT_LIST_DIR}/api.lua
```

Comment:
should this be CMAKE_CURRENT_SOURCE_DIR?

Reply:
Changed. I'm not pretty sure which one is preferred. In this case both are doing the same. Any explanation or is this a simple "best practice"?

Reply:
I would just say CMAKE_CURRENT_SOURCE_DIR is the much more commonly used of the two, whereas CMAKE_CURRENT_LIST_DIR seems more for cases where a file is `include`ed, instead of being part of a subdirectory of a build. But I don't have any actual rules which one to prefer.